ckanext/ontario_theme/resource.py

```diff
@@ -0,0 +1,190 @@
+u'''
+Overrides ckan/views/resource.py
+'''
+
+from ckan.views.resource import CreateView as CreateView
+from flask.views import MethodView
+import ckan.model as model
+import six
+import ckan.lib.datapreview as lib_datapreview
+import ckan.plugins as plugins
+import cgi
+
+from ckan.common import _, g, request
+from ckan.views.dataset import (
+    _get_pkg_template, _get_package_type, _setup_template_variables
```

Comment:
`_get_package_type, _setup_template_variables` are both imported but never used, we can remove them